plugins/linux-capture/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ set(linux-capture_LIBRARIES
 
 option(ENABLE_PIPEWIRE "Enable PipeWire support" ON)
 if(ENABLE_PIPEWIRE)
-	find_package(PipeWire QUIET)
+	find_package(PipeWire 0.3.40 QUIET)
```

Comment:
This will regress on our ubuntu packages which still support pipewire.

Reply:
A minimal version to build against is only required in the follow up #5131. Removed them from this MR and let's discuss them later. This MR should cover the pre 0.3.24 and the 0.3.24 and later state.